packages/bridge-ui-v2/config/schemas/configuredBridges.schema.json

```diff
@@ -41,6 +41,7 @@
             "required": [
               "bridgeAddress",
               "erc20VaultAddress",
+              "etherVaultAddress",
```

Comment:
why do we need this here? cc @KorbinianK

Reply:
"addresses": {
            "type": "object",
            "properties": {
              "bridgeAddress": {
                "type": "string"
              },
              "erc20VaultAddress": {
                "type": "string"
              },
              "etherVaultAddress": {
                "type": "string"
              },
              "erc721VaultAddress": {
                "type": "string"
              },
              "erc1155VaultAddress": {
                "type": "string"
              },
              "crossChainSyncAddress": {
                "type": "string"
              },
              "signalServiceAddress": {
                "type": "string"
              }
            },
            "required": [
              "bridgeAddress",
              "erc20VaultAddress",
              **"etherVaultAddress",**
              "erc721VaultAddress",
              "erc1155VaultAddress",
              "crossChainSyncAddress",
              "signalServiceAddress"
            ],
              I thought it should be there due to the code sequence at the top, but if this addition doesn't make sense, I can correct it.